docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,162 @@
+.. _acsincome-data:
+ACSIncome
+-------------------------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The ACSIncome dataset is one of five datasets created by Ding et al. [1]_ 
+as an improved alternative to the popular UCI Adult dataset [2]_.
+Briefly, the UCI Adult dataset is commonly used as a benchmark dataset when comparing
+different algorithmic fairness interventions. ACSIncome offers a few improvements,
+such as providing more datapoints (1,664,500 vs. 48,842) and more recent data (2018 vs. 1994).
+Further, the binary labels in the UCI Adult dataset indicate whether an indivdual earned 
+more than $50k US dollars in that year. Ding et al. show that the choice of threshold impacts
+the amount of fairness violation, so they allow users to define any threshold.
```

Comment:
I'd suggest adding at the end: "rather than fixing it at $50k." to be super explicit.

Reply:
Good addition!